modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1598 @@
+[source, cypher, role=test-fail]
+----
+MATCH (author:Author {name: 'Virginia Woolf'})
+REMOVE author.name
+----
+
+In this case, the property is not removed because it is in conflict with an existing constraint.
+
+.Error message
+[source, error]
+----
+Node(0) with label `Author` must have the property `name`
+----
+
+======
+
+
+.Remove a relationship property existence-constrained property
+======
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE CONSTRAINT wrote_locations FOR ()-[wrote:WROTE]-() REQUIRE wrote.location IS NOT NULL
+----
+////
+
+.Remove the `location` property from an existing `WROTE` relationship, given a property existence constraint `:WROTE(location)`
+[source, cypher, role=test-fail]
+----
+MATCH (author:Author)-[wrote:WROTE]->(book:Book)
+REMOVE wrote.location
+----
+
+In this case, the property is not removed because it is in conflict with an existing constraint.
+
+.Error message
+[source, error]
+----
+Relationship(0) with type `WROTE` must have the property `location`
+----
+
+======
+
+[[create-key-and-uniqueness-constraint-on-same-schema-as-existing-index]]
+==== Creating key and uniqueness constraints on the same schema as an existing index will fail
+
+
+.Creating a node property uniqueness constraint on the same name as an existing index
+======
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE INDEX preExisting_book_word_count FOR (book:Book) ON (book.wordCount)
+----
+////
+
+.Create a constraint requiring all `Book` nodes to have unique `wordCount` properties, when an index already exists on that label and property combination
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT book_word_count FOR (book:Book) REQUIRE book.wordCount IS UNIQUE
+----
+
+In this case, the constraint cannot be created because there already exists an index covering that schema.
+
+.Error message
+[source, error]
+----
+There already exists an index (:Book {wordCount}).
+A constraint cannot be created until the index has been dropped.
+----
+
+======
+
+
+.Creating a relationship property key constraint on the same name as an existing index
+======
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE INDEX knows FOR ()-[know:KNOW]-() ON (know.levels)
+----
+////
+
+.Create a constraint requiring all `KNOWS` relationships to have a unique level property as a relationship key, when an index already exists with the given name
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT knows
+FOR ()-[knows:KNOWS]-() REQUIRE (knows.level) IS RELATIONSHIP KEY
+----
+
+In this case, the constraint cannot be created because there already exists an index with the given name.
+
+.Error message
+[source, error]
+----
+There already exists an index called 'knows'.
+----
+
+======
+
+[[key-and-uniqueness-constraint-on-the-same-schema]]
+==== Key and uniqueness constraints are not allowed on the same schema
```

Comment:
surprise section, this was one I was missing when looking at the list above XD I guess it needs to be added to the list above then since it obviously exists XD